<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractionEditWindow</class>
 <widget class="QDialog" name="InteractionEditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Éditer une interaction</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Contenu de l'interaction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Saisir la description de l'interaction...
		
Chaque ligne peut contenir un &quot;@todo Description todo&quot; suivi optionnellement par un &quot;@date XX/XX/XXXX&quot; pour la date de réalisation. Si aucune date n'est spécifiée, alors le todo prend la date du jour.

Exemples :
@todo Faire le rapport @date 01/01/2021
@todo Faire les devoirs</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelDateCreation">
         <property name="text">
          <string>Date création : XX/XX/XXXX</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="buttonSupprimer">
         <property name="text">
          <string>&amp;Supprimer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonEnregistrer">
         <property name="text">
          <string>&amp;Enregistrer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonAnnuler">
         <property name="text">
          <string>&amp;Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>InteractionEditWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
